docs/literate/src/files/adding_new_equations_literate/cubic_conservation_law_literate.jl

```diff
@@ -0,0 +1,235 @@
+
+# We wrap the return value of the `initial_condition_sine` inside an `SVector` since that's the approach
+# used in Trixi.jl also for systems of equations. We need to index the spatial coordinate `x[1]`,
+# since it is an `SVector` with one component. In multiple space dimensions, all spatial coordinates
+# are passed together.
+  
+# The `ode` is an `ODEProblem` from the SciML/DifferentialEquations ecosystem.
+# Thus, we can solve this ODE numerically using any time integration method,
+# e.g., `SSPRK43` from [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl).
+# Before, we set up a [callback](@trixi-docs:callbacks/#callbacks-id) to summarize the simulation setup.
+summary_callback = SummaryCallback()
+callbacks = CallbackSet(summary_callback)
+  
+## OrdinaryDiffEq's `solve` method evolves the solution in time and executes the passed callbacks
+sol = solve(ode, SSPRK43(),
+            save_everystep=false, callback=callbacks, maxiters=1e5);
+  
+## Print the timer summary
+summary_callback()
+#src # TODO summary_callback() is printing summary automatically (even with ;)
+
+# That's it, you ran your first simulation using your new equation with Trixi! Now,
+# we can plot the solution at the final time using Plots.jl.
+
+using Plots
+plot(sol)
+#src # ![tutorial_adding_new_equations_plot1](https://user-images.githubusercontent.com/12693098/111651488-91122980-8806-11eb-848c-af09f3af234c.png)
+#src # ![tutorial_adding_new_equations_plot1b](https://user-images.githubusercontent.com/74359358/126663757-b8fb3ecb-72c8-4246-ab64-2bb4dbdf621a.png)
+
+# You can already see that discontinuities will develop and oscillations start to
+# occur around steep parts of the wave. That's expected from our central discretization.
+# To avoid these issues, we need to use dissipative numerical fluxes (approximate
+# Riemann solvers) at interfaces.
+  
+  
+# ## Advanced setup
+  
+# Thus, we add a Godunov's flux for our cubic equation. That is easy for this equation
+# since the wave speed `f'(u) = 3u^2` is always non-negative.
+
+@inline Trixi.flux_godunov(u_ll, u_rr, orientation, equation::CubicConservationLaw.CubicEquation) = flux(u_ll, orientation, equation)
+
+# Let's run the example again but with a dissipative numerical flux at interfaces.
+# `remake` will recreate the semidiscretization we used before and only change
+# selected parameters, in this case the `solver`.
+
+## A new setup with dissipation
+semi = remake(semi, solver=DGSEM(3, flux_godunov))
+ode = semidiscretize(semi, tspan)
+sol = solve(ode, SSPRK43(),
+            save_everystep=false, callback=callbacks, maxiters=1e5);
+summary_callback()
+plot!(sol)
+#src # ![tutorial_adding_new_equations_plot2](https://user-images.githubusercontent.com/12693098/111651740-c9196c80-8806-11eb-9a02-c0420eecf4fc.png)
+#src # ![tutorial_adding_new_equations_plot2b](https://user-images.githubusercontent.com/74359358/126664312-e2f2fc93-42b0-4082-a009-b4268b15ff21.png)
```

Comment:
Two plots?

Reply:
None of these plots are used right now. The notebooks have no saved output and the markdown are executed such that we plots are generated automatically.
I just kept the link here for possible changes in the future.
There are two plots because the first one (2; from the original Trixi) is slightly different but maybe a bit more expressive than the executed plot (2b).

Reply:
Okay. Could you please describe this in a comment above these lines?